### DISCUSSION TITLE
Switching between tabs on gnome doesnt work

### DISCUSSION THREAD

**Discussion by yotam5** [wezterm/wezterm#2072] [Q&A] [answered]
hey im new to wezterm and currently learning the terminal and how to change configurations, but i cant manage to switch between tabs, i can activate tab but the text of the activated tab wont show, how do i switch between tabs? thanx

**Comment by wez** [comment]
Please share more precise details about what you tried and what you saw; right now it's hard to know what to suggest because there isn't much information in your post.

**Reply by yotam5** [reply]
sure thing my mistake,

Os: 
opensuse x86_64 thumbwheel

What i tried: 
to switch between tabs. i open two tabs and in one of them i executed "ls" command, i tried to switch to the empty one using
CTRL+ ALT + N when n is the number of the tab, from what i understand its how to switch between tabs

What i saw:

well, it selected the tab i chose and highlighted it BUT the text is still as in the previous one, so it highlighted the empty tab but DIDNT SWITCH to it cuz the text showing on the terminal is still the tab that i ran the "ls" command

maybe i tried the wrong shortcut or i missed something is the conf that effect the outcome, this is my current conf:

local wezterm = require 'wezterm';

local mykeys = {}
for i = 1, 8 do
  -- CTRL+ALT + number to move to that position
  table.insert(mykeys, {
    key=tostring(i),
    mods="CTRL|ALT",
    action=wezterm.action{MoveTab=i-1},
  })
end

return {
  font = wezterm.font("JetBrains Mono"),
  font_size = 18.0,
  harfbuzz_features = { 'calt=0', 'clig=0', 'liga=0' },
  color_scheme = "Dark+",
  hide_tab_bar_if_only_one_tab = true,
  keys = mykeys,
}

**Reply by wez** [reply]
It's possible that your window manager has predefined shortcuts for `CTRL+ALT+number`; I would suggest checking to see if that is true.

The default key assignment for switching tabs are `CTRL+SHIFT+number` and `SUPER+number` (see: https://wezfurlong.org/wezterm/config/default-keys.html).

Turning on https://wezfurlong.org/wezterm/config/lua/config/debug_key_events.html can be helpful in figuring out whether wezterm is receiving a key event from the WM and seeing how it is interpreted.

**Reply by yotam5** [reply]
it was a bit confusing but i understood, it switched between the tabs places[1,2] to [2,1] because of my settings 
you can close, thanx

**Reply by wez** [reply]
[MoveTab](https://wezfurlong.org/wezterm/config/lua/keyassignment/MoveTab.html) is intended to be used to move the active tab index to the provided tab index. It doesn't switch the active tab contents.
Do you want [ActivateTab](https://wezfurlong.org/wezterm/config/lua/keyassignment/ActivateTab.html) instead?

**Reply by yotam5** [reply]
yes it will fix the CTRL+ALT+N, to fix the SUPER+N in gnome you need to disable the default keybinding which is hidden from the setting in the GUI

**Comment by yotam5** [comment, marked as answer]
solution:

the problem is that gnome has keybindings that are not in the settings CNTRL+1 launches firefox for example, can be fixed with this:

for i in $(seq 1 9)
do
  gsettings set org.gnome.shell.keybindings switch-to-application-$i '[]'
done
